person_capture/gui_app.py

```
@@ -8281,6 +8558,117 @@ def _quantile_lut(src_ch: np.ndarray, dst_ch: np.ndarray, mask: np.ndarray) -> O
+        base = out_path + ".tmp_yuv444_fast"
+        ref_hdr = base + f".ref_{ref_w}x{ref_h}.avif"
+        ref_png = base + f".ref_{ref_w}x{ref_h}.png"
+        clean_hdr = base + ".clean_yuv444.avif"
+        clean_png = base + ".clean_yuv444.png"
+        repaired_tmp = ""
+        try:
+            for pth in (ref_hdr, ref_png, clean_hdr, clean_png):
+                if os.path.exists(pth):
+                    os.remove(pth)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)